src/global.css

```diff
@@ -1,3 +1,23 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+input[type='checkbox']::before {
+  content: '';
+  width: 0.65em;
+  height: 0.65em;
+  transform: scale(0);
+  transition: 60ms transform ease-in-out;
+  box-shadow: inset 1em 1em #fff;
+  transform-origin: bottom left;
+  clip-path: polygon(14% 44%, 0 65%, 50% 100%, 100% 16%, 80% 0%, 43% 62%);
+  cursor: pointer;
+}
+
+input[type='checkbox']:checked::before {
+  transform: scale(1);
+}
+
+.modal-box {
+  @apply p-8;
+}
```

Comment:
`@apply` is one of the strategies for [overriding the DaisyUI default theme](https://daisyui.com/docs/customize/) without creating your own.

Reply:
Very cool! Didn't know you could do that.